cypress/e2e/tests/migration/dynamic-report/dynamic-report.ts

```diff
@@ -0,0 +1,90 @@
+import {
+    click,
+    clickByText,
+    inputText,
+    performWithin,
+    selectFilter,
+    selectItemsPerPage,
+    selectUserPerspective,
+    waitUntilSpinnerIsGone,
+} from "../../../../utils/utils";
+import { button, issueFilter, migration, SEC, singleApplication } from "../../../types/constants";
```

Comment:
I think you incorrectly placed this file under tests instead of models.

Reply:
right, thanks for noticing that